externs/shaka/manifest.js

```diff
@@ -60,6 +61,9 @@
  * @property {!Array.<shaka.extern.Stream>} textStreams
  *   <i>Required.</i> <br>
  *   The presentation's text streams.
+ * @property {!Array.<shaka.extern.Stream>} imageStreams
+ *   <i>Required.</i> <br>
+ *   The presentation's image streams
```

Comment:
I'm curious - are images in the stream only I-frames or something like that?

Reply:
I-Frames is a video track that contains a single frame, but in a video container. Thumbnails are images in image format (eg: jpg, png, etc.)

Reply:
Yeah, I meant like, is there one for every second or so?